lib/util.js

```diff
@@ -53,6 +53,22 @@ export const allProfiles = [
     ...filesSUBM.map(x => `SUBM/${x}`),
 ];
 
+export const profiles = Object.fromEntries(
+    allProfiles
+        .map(file => {
+            const match =
+                /((TR|SUBM)\/([A-Za-z]+\/)?([A-Z][A-Z-]*[A-Z](-Echidna)?))\.js$/.exec(
+                    file
+                );
+            if (match && match[4]) {
+                const key = match[4];
+                return [key, import(`./profiles/${match[0]}`)];
```

Comment:
Not sure if it should be `match[0]` or `match[1]`, but rest looks good! `match[0]` is same as `file`, right?